configurator.js

```diff
@@ -0,0 +1,170 @@
+/* 
+ * Configurator: Handles configuration for the Radar server. 
+ *
+ * It provides the default configuration.
+ * For each allowed variable, it provides a default, which can be overwritten 
+ * through ENV or ARGV. 
+ * 
+ * All the knowledge of what comes in, belongs here. 
+ * 
+ * ARGV > ENV > DEFAULTS
+ *
+ */
+
+var variables = [
+  {
+    name:     'port', description: 'port to listen',
+    env:      'RADAR_PORT', 
+    abbr:     'p',
+    full:     'port',
+    default:  '8000'
+  },
+  {
+    name:     'redisUrl', description: 'Redis url',
+    env:      'RADAR_REDIS_URL',
+    abbr:     'r',
+    full:     'redis_url',
+    default:  'redis://localhost:6379'
+  },
+  {
+    name:     'healthReportInterval', description: 'healthReportInterval',
+    env:      'RADAR_HEALTH_REPORT_INTERVAL',
+    full:     'interval',
+    abbr:     'i',
+    default:  '10000'
+  },
+  {
+    name:     'sentinelMasterName', description: 'sentinel master name',
+    env:      'RADAR_SENTINEL_MASTER_NAME', 
+    full:     'sentinel_master_name'
+  },
+  {
+    name:     'sentinelUrls', description: 'sentinel urls',
+    env:      'RADAR_SENTINEL_URLS',
+    full:     'sentinel_urls'
+  }
+];
+
+function defaultConfiguration() {
+  var config = {};
+
+  variables.forEach(function(element) {
+    if (element.hasOwnProperty('default')) {
+      config[element.name] = element.default;
+    }
+  });
+
+  return config;
+}
+
+function parserCli(argv) {
+  var parser = require('nomnom')();
+  
+  variables.forEach(function(element){
+    parser.option(element.name, {
+      help:     element.description, 
+      full:     element.full,
+      abbr:     element.abbr
+    });
+  });
+
+
```

Comment:
Sometimes you have 1 line of space before *return*, sometimes 2, as here; let's make it 1.